rasa/cli/__init__.py

```diff
@@ -0,0 +1,5 @@
+import os
+import sys
+
+# insert current path in syspath so module is found
+sys.path.insert(1, os.getcwd())
```

Comment:
I think it would be safer if we put it in `main()` of the module `__main__`. What do you think?

Reply:
yeah that seems to make more sense -- will be easier for others to notice if they're debugging something funky that happens 👍